src/WebJobs.Script/Description/Node/Script/functionTemplate.js

```diff
@@ -40,4 +42,36 @@ return function (context, callback) {{
     delete context.inputs;
 
     f.apply(null, inputs);
-}};
\ No newline at end of file
+}};
+
+function getEntryPoint(f, context) {{
+    if (util.isObject(f)) {{
```

Comment:
This entire file is a string template, so they're escaped :) I made a comment elsewhere that as this file grows, the need to move the fx bits out into a proper node module become more apparent. That way I can add some Mocha tests to the  repo. Right now, I rely on e2e tests.